libexec/bats-core/bats

```diff
@@ -33,6 +33,10 @@ HELP_TEXT_HEADER
                             A two character string of code quote delimiters
                             or 'custom' which requires setting $BATS_BEGIN_CODE_QUOTE and 
                             $BATS_END_CODE_QUOTE. Can also be set via $BATS_CODE_QUOTE_STYLE
+  -e, --exclude <files>     Exclude file(s) from processing. File can also be a
+                            directory itself. <files> are separated by space 
+                            and enclosed in quotes, e.g. 
+                            `--exclude 'foo.bats bar baz qux.bats'`
```

Comment:
Maybe we should go for multiple --exclude statements to allow for spaces in filenames (although that is hopefully rare).

Reply:
i prefer one exclude statement
we could introduce an escape character for spaces.